sites/public/src/pages/applications/review/terms.tsx

```diff
@@ -20,14 +20,22 @@ import {
   MultiselectQuestionsApplicationSectionEnum,
 } from "@bloom-housing/shared-helpers/src/types/backend-swagger"
 import ApplicationFormLayout from "../../../layouts/application-form"
-import { Button } from "@bloom-housing/ui-seeds"
+import { Button, Dialog } from "@bloom-housing/ui-seeds"
 
 const ApplicationTerms = () => {
   const router = useRouter()
   const { conductor, application, listing } = useFormConductor("terms")
-  const { applicationsService, profile } = useContext(AuthContext)
+  const { applicationsService, authService, loadProfile, profile } = useContext(AuthContext)
   const [apiError, setApiError] = useState(false)
   const [submitting, setSubmitting] = useState(false)
+  const [sessionVoided, setSessionVoided] = useState(false)
+
+  const closeCallback = useCallback(() => {
+    setSubmitting(false)
+    conductor.reset()
+    void router.push("/sign-in")
```

Comment:
Issue: When clicking the close button I was redirected to the home page rather than the sign-in page. I'll keep investigating this

Reply:
Update: this has been fixed